include/OledDisplay.h

```diff
@@ -0,0 +1,21 @@
+#ifndef OLEDDISPLAY_H
+#define OLEDDISPLAY_H
+
+#include <U8g2lib.h>
+
+#define SCL 22
+#define SDA 21
+
+class Display {
+  private:
+    U8G2_PCD8544_84X48_1_4W_HW_SPI _nokia = U8G2_PCD8544_84X48_1_4W_HW_SPI(U8G2_R0, 5, 4, 16);
+    U8G2_SSD1306_128X64_NONAME_F_HW_I2C _ssd1306 = U8G2_SSD1306_128X64_NONAME_F_HW_I2C(U8G2_R0, SCL, SDA, U8X8_PIN_NONE);
+    U8G2_SH1106_128X64_NONAME_F_HW_I2C _sh1106 = U8G2_SH1106_128X64_NONAME_F_HW_I2C(U8G2_R0, SCL, SDA, U8X8_PIN_NONE);
+    int display_type;
+  public:
```

Comment:
Bitte wie in allen anderen klassen zuerst die public methoden, dann die private methoden